src/node/rpc/frontend.h

```diff
@@ -218,66 +218,25 @@ namespace ccf
 
       Store::Tx tx;
 
-      // Retrieve id of caller
-      std::optional<CallerId> caller_id;
-      if (ctx->is_create_request)
```

Comment:
No need for this special case anymore: just mark the genesis request as non authenticated.